integration-tests/api/__tests__/admin/variant.js

```diff
@@ -145,4 +147,124 @@ describe("/admin/products", () => {
       )
     })
   })
+
+  describe("GET /admin/variants price selection strategy", () => {
+    beforeEach(async () => {
+      await storeProductSeeder(dbConnection)
+      await adminSeeder(dbConnection)
+
+      await simpleProductFactory(
+        dbConnection,
+        {
+          title: "prod",
+          variants: [
+            {
+              title: "test1",
+              inventory_quantity: 10,
+            },
+            {
+              title: "test2",
+              inventory_quantity: 12,
+            },
+          ],
+        },
+        100
+      )
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      await db.teardown()
+    })
+
+    it("selects prices based on the passed currency code", async () => {
```

Comment:
suggestion: We are only testing for the successful / expected case that is we've created a variant with id `test-variant` that only has prices in USD. 

To cover more cases, we should use a variant with prices in multiple different currencies.